Code/positionnement/Funibot.cpp

```diff
@@ -38,9 +38,13 @@ void Funibot::addPole(FuniMath::Vecteur positionPole, FuniMath::Vecteur position
 	// Dépassement du nombre de pôles maximum
 	if (nbrPole == FuniConst::NBR_POLES)
 	{
-		throw 1;
+		GestionErreurs::erreur erreur;
+		erreur.id = 1;
+		TIMESTAMP_ERREUR(erreur);
+		erreurs.addBack(erreur);
+		return;
```

Comment:
Je remplacerais la verbosité de création des erreurs par une méthode du genre:
```suggestion
		erreurs.emplace(1, TIMESTAMP());
		return;
```
Avec l'id et le temps comme paramètres directement.
Ici, la macro TIMESTAMP() callerait millis() sur Arduino et rien ou un équivalent STD sur un PC, mettons.

Reply:
Sinon, tu peux utiliser la fonctionnalité d'initialisation de struct pour faciliter:
![image](https://user-images.githubusercontent.com/55722645/107299012-86fd4e00-6a44-11eb-9905-b59e20fc7f92.png)
Pas sûr que ça marche sur Arduino par contre.